src/pages/trader-tools/_validation.js

```diff
@@ -39,7 +40,7 @@ const numberValidation = (input, fieldName, maxDigit) => {
 
 const numberWithNegativeValidation = (input, fieldName, maxDigit) => {
     if (!input) {
-        return localize(`${fieldName} is required`)
+        return <Localize translate_text="{{fieldName}} is required" values={{ fieldName }} />
```

Comment:
```suggestion
        return <Localize translate_text="{{fieldName}} is required" values={fieldName} />
```

Reply:
The syntax should have double `{{` or else it will not extracted